esp/services/ws_fileio/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ include_directories (
          ./../../smc/SMCLib 
          ./../../bindings/SOAP/xpp 
          ${HPCC_SOURCE_DIR}/common/thorhelper
+         ${HPCC_SOURCE_DIR}/dali/sasha
```

Comment:
This suggests the header files are not very clean.  Better would be for ISashaCommand to be an opaque type.